conf/modules.conf

```diff
@@ -1589,7 +1589,7 @@
 # SASL authentication module: Provides support for IRC Authentication
 # Layer via AUTHENTICATE. Note: You also need to have m_cap.so loaded
 # for SASL to work.
-<module name="m_sasl.so">
+#<module name="m_sasl.so">
```

Comment:
Is there is reason why we no longer load `m_sasl`?

Reply:
In [v2 SASL](https://docs.inspircd.org/2/modules/sasl/), `target` is not required, but in [v3 SASL](https://docs.inspircd.org/3/modules/sasl/) it is now required...So if insp3 is not configured with a services server, then there's no way to pass the now-required target to the SASL module config.

I just made a [PR](https://github.com/inspircd/inspircd-docs/pull/8) on the documentation.

It's somewhat kludgy, but we can add
```
<module name="m_sasl.so">
<sasl target="$INSP_SERVICES_NAME">
```
to `services.sh`, so that the SASL module only gets enabled if a services server is configured. That way people can still have services and SASL without having to write a custom config.